modules/installation-gcp-permissions.adoc

```diff
@@ -335,54 +36,26 @@ endif::ipiupi[]
 ifdef::template[]
 .Required roles for user-provisioned GCP infrastructure
 * Deployment Manager Editor
-* Service Account Key Admin
```

Comment:
@xenolinux Any reason why `Service Account Key Admin` was removed from **Required roles for user-provisioned GCP infrastructure**?

Reply:
@jianli-wei suggested so because `Service Account Key Admin` was listed in both places - "Required roles for user-provisioned GCP infrastructure" and "optional roles".

Reply:
@jianli-wei would it make sense to remove the "Optional roles" and add `Service Account Key Admin` back to **Required roles for user-provisioned GCP infrastructure**?

Reply:
@arkadeepsen I think @jianli-wei had suggested the same.

Reply:
I mistakenly removed from required....

Reply:
Addressed as suggested.

Reply:
@xenolinux I think you need to add ifeval for IPI separately for `Optional roles`. Also `Required roles for user-provisioned GCP infrastructure` should not be displayed for IPI. The existing ifdef should not be removed.

Reply:
Addressed